sos/report/plugins/ceph_podman.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2020 Meiyan Zheng <mzheng@redhat.com>
+#
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+import re
+
+
+class CephPodman(Plugin, RedHatPlugin):
+    short_desc = 'Containerized Ceph distributed storage'
+    plugin_name = 'ceph_podman'
+    profiles = ('storage', 'container')
+    packages = ('podman',)
+
+    def setup(self):
+        self._container_mon_name = self.get_container_by_name('ceph-mon-*')
+        self._container_osd_name = self.get_container_by_name('ceph-osd-*')
+
+        self.add_copy_spec([
+            "/etc/ceph/",
+            "/var/lib/ceph/",
+            "/run/ceph/"
+        ])
```

Comment:
These would still be captured from the host, not inside the container. I'm not sure if that's what you want here or not though.

Reply:
Those files are mounted from host to container, so I guess it's fine to capture those files from host: 

# podman inspect ceph-mon-overcoud-controller-0 | jq .[].HostConfig.Binds
[
  "/var/lib/ceph:/var/lib/ceph:rw,rprivate,rbind",
  "/etc/ceph:/etc/ceph:rw,rprivate,rbind",
  "/var/run/ceph:/var/run/ceph:rw,rprivate,rbind",
  "/etc/localtime:/etc/localtime:ro,rprivate,rbind",
  "/var/log/ceph:/var/log/ceph:rw,rprivate,rbind",
  "/etc/pki/ca-trust/extracted:/etc/pki/ca-trust/extracted:rw,rprivate,rbind"
]

# podman inspect ceph-mgr-overcloud-controller-0 | jq .[].HostConfig.Binds
[
  "/var/log/ceph:/var/log/ceph:rw,rprivate,rbind",
  "/var/lib/ceph:/var/lib/ceph:rw,rprivate,rbind",
  "/etc/ceph:/etc/ceph:rw,rprivate,rbind",
  "/var/run/ceph:/var/run/ceph:rw,rprivate,rbind",
  "/etc/localtime:/etc/localtime:ro,rprivate,rbind"
]